cli/tsc/99_main_compiler.js

```diff
@@ -9,7 +9,7 @@
 // instances convert TypeScript to JavaScript.
 
 // Removes the `__proto__` for security reasons.  This intentionally makes
-// Deno non compliant with ECMA-262 Annex B.2.2.1
+// Deno non compliant with ECMA-262 20.1.3.8
```

Comment:
How is Deno non-compliant if the `__proto__` accessor is normative optional?

Reply:
oops! How about just removing it? @littledivy 
```suggestion
```

Reply:
that works!

Reply:
I've fixed this and decided not to change comment in testdata! thanks :) @littledivy